src/components/YourCompany/YourCompany.vue

```diff
@@ -173,15 +173,27 @@
           </div>
         </v-col>
       </v-row>
+    </div>
 
-      <!-- Business Type -->
-      <correct-business-type class="sub-section"
+    <!-- Business Type -->
+    <div v-if="isAlterationView"
+         id="company-type-section"
+         class="section-container"
+         :class="{'invalid-section': invalidTypeSection}"
+    >
+      <changeBusinessType
```

Comment:
Should be capital C.

You should try VS Code. It nicely highlights components (if they follow CamelCase naming).

Reply:
oops, i thought it was lower case in the template

Never!! Love my webstorm haha

Reply:
fixed